test/models/pwa.js

```diff
@@ -20,6 +20,7 @@ const assert = require('assert');
 const Manifest = require('../../models/manifest');
 const Pwa = require('../../models/pwa');
 
+/* eslint-disable camelcase */
```

Comment:
This is not about coding standards, but about readability (I wouldn't disable all rules). In this particular test file most tests that create a PWA will have to disable the camelcase check. This is why I'd prefer to do it once at the top...